components/member_list_team/member_list_team.tsx

```diff
@@ -58,14 +59,13 @@ export default class MemberListTeam extends React.Component<Props, State> {
         };
     }
 
-    componentDidMount() {
-        this.props.actions.loadProfilesAndTeamMembers(0, Constants.PROFILE_CHUNK_SIZE, this.props.currentTeamId).then(({data}) => {
-            if (data) {
-                this.loadComplete();
-            }
-        });
-
-        this.props.actions.getTeamStats(this.props.currentTeamId);
+    async componentDidMount() {
+        await Promise.all([
+            this.props.actions.loadProfilesAndTeamMembers(0, Constants.PROFILE_CHUNK_SIZE, this.props.currentTeamId),
+            this.props.actions.getTeamMembers(this.props.currentTeamId),
+            this.props.actions.getTeamStats(this.props.currentTeamId),
+        ]);
+        this.loadComplete();
```

Comment:
Is the `if (data)` check in the old code still relevant?

Reply:
@lieut-data 
I looked at the function that was using it (https://github.com/mattermost/mattermost-webapp/blob/master/actions/user_actions.jsx#L38) and it seems like data is always returned as true so I thought it's better to leave it out and not use it as it doesn't seem to have any purpose.